### DISCUSSION TITLE
nonebot.adapters.onebot.v11.exception.NetworkError: <NetWorkError message=WebSocket API call timeout>

### DISCUSSION THREAD

**Discussion by songdf666** [nonebot/discussions#132] [问答] [answered]
在调用cqhttp的发送信息的API时偶尔会出现以下问题
![image](https://user-images.githubusercontent.com/81544526/166104844-5e3a2185-91bf-42e0-8c6d-d6f2cf295a1f.png)
可能是cqhttp连接超时？因为经常需要在cqhttp页面回车一下...求大佬指点如何解决！！！谢谢🙏
help

**Comment by mnixry** [comment, marked as answer]
别用鼠标点窗口中间... 会暂停进程运行的
如果只是偶尔出现可以无视)

**Reply by songdf666** [reply]
谢谢您！很荣幸能够得到您的答复，这种情况大概一天会出现4-5次，555，别用鼠标点窗口中间您说的是哪个窗口，cqhttp的窗口嘛？诚盼您的回复！祝好！

**Reply by mnixry** [reply]
> 这种情况大概一天会出现4-5次

如果消息量比较大的话其实这个概率算是比较低的, 基本可以无视

你也可以尝试增加[`API_TIMEOUT`](https://v2.nonebot.dev/docs/next/tutorial/configuration#api-timeout)的值来降低这个出现的可能性

> 别用鼠标点窗口中间您说的是哪个窗口，cqhttp的窗口嘛？

对 点了以后窗口应该标题会变成 `选择: xxx` 这样的, 可以注意看一下

**Reply by Skyminers** [reply]
我也有相同的问题，在发送较大图片时基本上 100% 触发这个问题

**Reply by mnixry** [reply]
> 我也有相同的问题，在发送较大图片时基本上 100% 触发这个问题

~~那有可能是你网太慢了~~ 可能是上传时间过长导致的超时， 你可以尝试增大[`API_TIMEOUT`](https://v2.nonebot.dev/docs/next/tutorial/configuration#api-timeout)
